### ISSUE TITLE
Option to Open Emulators in Emudeck like in RetroBat

### ISSUE THREAD

**Issue by d5364t54ytfr4** [dragoonDorise/EmuDeck#825]
Currently to open Emulators I have to go to Steam and Steam when not in Big Picture mode is hard to navigate on the ROG ally. Or I have to go to the File Manager which is also a PITA in Ally.

I might need this for changing the settings of the emulator for example and EmuDeck makes this somewhat hard currently as mentioned above.

But I tested RetroBat and it has a menu along with the other menus for platforms just called RetroBat and when you open the menu instead of games, you see all the emulators which you can open like you would games.

TL;DR: Retrobat has a menu RetroBat along with the other platform menus(Like SNES, NES etc) and when clicked shows emulators to launch directly instead of games. Useful for directly opening emulators and things like configuring the emulator etc.
Hope this feature can be added.

**Comment by Godsbane** [comment]
We have this already. It's the Emulators parser in SRM.
